clang/lib/Sema/SemaTemplateInstantiateDecl.cpp

```diff
@@ -6148,6 +6148,35 @@ NamedDecl *Sema::FindInstantiatedDecl(SourceLocation Loc, NamedDecl *D,
   return D;
 }
 
+static void processSYCLKernel(Sema &S, FunctionDecl *FD, MangleContext &MC) {
+  // Because all SYCL kernel functions are template functions - they
+  // have deferred instantination. We need bodies of these functions
+  // so we are checking for SYCL kernel attribute after instantiation.
```

Comment:
I think this comment became out of context here. Lets remove it.

Reply:
Done.